webnode/src/utils/datamap.js

```diff
@@ -2,8 +2,8 @@ import _ from "lodash";
 import iota from "redux/services/iota";
 import Encryption from "utils/encryption";
 
-const rawGenerate = (handle, size) => {
-  const keys = _.range(0, size + 1);
+const rawGenerate = (genesisHash, size) => {
+  const keys = _.range(1, size);
```

Comment:
We can undo the changes I've made to this method if we want to, I have no strong feelings about it.  Basically what this means is that now when you tell it a genesis hash and 50 chunks for example, you will get back exactly 50 hashes and the genesis hash will be the first in the array.  This was so that when a treasure was at idx 1 in the actual datamap, it didn't show up as idx 0 in the treasure hunting UI.